### ISSUE TITLE
What are the visual attributes considered here?

### ISSUE THREAD

**Issue by ParikhKadam** [msracver/Deep-Image-Analogy#45]
I am currently reading the paper and confused on what is actually considered as an attribute to transfer. It lists three things: color transfer, texture transfer, and style transfer.

In the example image of monalisa and avatar, looking at the image A', I made the following observation:
- it transferred the color, eyes and lips.

Then why didn't it transfer ears and nose?

What I understand is, it should only transfer color in this set of images. Eyes, lips, ears and nose are individual attributes of that person itself and it shouldn't change. Yes, it may be able to transfer style, i.e. if B' is a photo of girl with tiara headband, then it should transfer this style to A producing A'. But this isn't the case here..

In short, are we actually controlling the attributes to transfer? Or the model takes decisions on its own?

Thank you..

**Comment by rozentill** [comment]
It does transfer ears and nose, but distort it to match the content image's structure. You can definitely change the weights to control the distortion to get a more avatar-like ear or nose.
